howso/upgrade.amlg

```diff
@@ -330,42 +357,42 @@
 							(parse (get import_metadata_map "!featureCustomDerivedMethods"))
 						)
 				))
-			)
-		)
-
-		;overwrite this trainee code with the new code, this does not overwrite contained entities (cases, sessions),
-		;but does overwrite all labels including versions with new ones
-		(assign_entity_roots (retrieve_entity_root new_trainee_path))
-		(destroy_entities new_trainee_path)
-
-		;for each label pull its value from the old trainee and overwrite it in the new one
-		(assign_to_entities import_metadata_map)
 
-		;import cases and sessions from saved json and re-create them in the trainee
-		(if import_cases_and_sessions_map
-			;iterate over all the entities and re-create them as contained entities with corresponding features and values
-			(map
-				(lambda
-					(create_entities
-						(current_index)
-						(set_type
-							(zip_labels
-								(indices (current_value))
-								(values (current_value))
+				;overwrite this trainee code with the new code, this does not overwrite contained entities (cases, sessions),
+				;but does overwrite all labels including versions with new ones
+				(assign_entity_roots (retrieve_entity_root new_trainee_path))
```

Comment:
is this step even necessary since the 'new trainee' is what's doing the upgrading and for subtrainees, those are also clones of new_trainee_path already (from the clone_entities line inside !UpgradeFromSubTrainee)?

Reply:
yeah I think you're right